src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/model/Utils.java

```diff
@@ -33,4 +43,27 @@ public static boolean isNetworkAvailable(final Context context) {
         return connectivityManager.getActiveNetworkInfo() != null && 
             connectivityManager.getActiveNetworkInfo().isConnectedOrConnecting();
     }
+
+    public static void sendFeedEvent(Context context, String category) {
+        Log.d(TAG, "Logging feed event. Category is " + category);
+
+        getTracker(context).send(new HitBuilders.EventBuilder()
+                .setCategory(category)
+                .setAction("click")
+                .build());
+    }
+
+    private static Tracker getTracker(Context context) {
+        if (tracker == null) {
```

Comment:
Sounds good on the application context @atavism. With the mobile SDK code, we are creating two analytics objects for the same tracking ID now though right or no? Seems like that could create some confusing data if so.